finals-2024-01-15/code/5029-angel-nineski/src/app/services/movie-data.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Movie } from '../models/movie';
+import { ActivatedRoute } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MovieDataService {
+  private apiUrl = 'db/movie-data.json';
```

Comment:
This `apiUrl` is wrong, as it works with the json file instead of a backend. Anything else than the basic `get` call will not work.